examples/models/llama/config/llm_config_utils.py

```diff
@@ -26,92 +26,143 @@ def convert_args_to_llm_config(args: argparse.Namespace) -> LlmConfig:
     llm_config = LlmConfig()
 
     # BaseConfig
-    llm_config.base.model_class = ModelType(args.model)
-    llm_config.base.params = args.params
-    llm_config.base.checkpoint = args.checkpoint
-    llm_config.base.checkpoint_dir = args.checkpoint_dir
-    llm_config.base.tokenizer_path = args.tokenizer_path
-    llm_config.base.metadata = args.metadata
-    llm_config.base.use_lora = bool(args.use_lora)
-    llm_config.base.fairseq2 = args.fairseq2
+    if hasattr(args, "model"):
+        llm_config.base.model_class = ModelType(args.model)
+    if hasattr(args, "params"):
+        llm_config.base.params = args.params
+    if hasattr(args, "checkpoint"):
+        llm_config.base.checkpoint = args.checkpoint
+    if hasattr(args, "checkpoint_dir"):
+        llm_config.base.checkpoint_dir = args.checkpoint_dir
+    if hasattr(args, "tokenizer_path"):
+        llm_config.base.tokenizer_path = args.tokenizer_path
+    if hasattr(args, "metadata"):
+        llm_config.base.metadata = args.metadata
+    if hasattr(args, "use_lora"):
```

Comment:
Can you put these in the PR when you first introduce `llm_config_utils.py`, so that we don't have to change it here.